stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -576,13 +576,13 @@ end
 # =============================================================================================
 
 #! Verifies a signature against a public key and a message. The procedure gets as inputs the hash
-#! of the public key and the hash of the message via the operand stack. The signature is provided
-#! via the advice stack.
+#! of the public key and the hash of the message via the operand stack. The signature is expected 
+#! to be present on the advice stack.
 #! The signature is valid if and only if the procedure returns.
 #!
 #! Inputs:
 #!   Operand stack: [PK, MSG, ...]
-#!   Advice stack:  [tau0, tau1, h_0, ..., h_511, s2_0, ..., s2_511, pi_0, ..., pi_1022, nonce0, ..., nonce7]
+#!   Advice stack:  [SIG, tau0, tau1, h_0, ..., h_511, s2_0, ..., s2_511, pi_0, ..., pi_1022, nonce0, ..., nonce7]
```

Comment:
I think `tau0, tau1, h_0, ..., h_511, s2_0, ..., s2_511, pi_0, ..., pi_1022, nonce0, ..., nonce7` here is the signature - so, `SIG` is redundant. Or if we add `SIG` we should remove the other elements.

Reply:
Switched to just `SIGNATURE`